src/Stratis.Bitcoin.Features.Miner/PowMining.cs

```diff
@@ -41,64 +56,69 @@ public class PowMining : IPowMining
         /// </summary>
         public const int DefaultBlockMaxWeight = 3000000;
 
-        private const int InnerLoopCount = 0x10000;
-
-        /// <summary>Manager of the longest fully validated chain of blocks.</summary>
-        private readonly IConsensusLoop consensusLoop;
-
-        private readonly ConcurrentChain chain;
+        private uint256 hashPrevBlock;
 
-        private readonly Network network;
+        private const int innerLoopCount = 0x10000;
```

Comment:
Can you please add the first capital letter? 

>CodingStyle: 12. We use PascalCasing to name all our constant local variables and fields.

Reply:
Fixed.